src/py/flwr/common/config.py

```diff
@@ -188,20 +188,17 @@ def parse_config_args(
     # Regular expression to capture key-value pairs with possible quoted values
     pattern = re.compile(r"(\S+?)=(\'[^\']*\'|\"[^\"]*\"|\S+)")
 
+    # If a .toml is passed, all other keys are ignored
```

Comment:
Instead of doing this, shouldn't we raise an error when there's a TOML file and also other keys?